cmd/createtree/main_test.go

```diff
@@ -218,6 +232,9 @@ func (s *fakeAdminServer) CreateTree(ctx context.Context, req *trillian.CreateTr
 		return nil, s.err
 	}
 	resp := *req.Tree
+	if req.KeySpec != nil && s.generatedKey != nil {
```

Comment:
In the past, I've gotten the impression that you'd rather avoid having test code that simply detects whether the test is misconfigured. However, I'm happy to add this if you'd prefer in this case.